src/api/frameworks/operationsCoverter.ts

```diff
@@ -0,0 +1,30 @@
+import { Stack, StackOutputOnlyFields } from "./interfaces";
+import * as poller from "../../operation-poller";
+import { frameworksOrigin } from "../../api";
+import * as rpc from "./rpcHandler";
+import { API_VERSION } from "./rpcHandler";
+
+const frameworksPollerOptions: Omit<poller.OperationPollerOptions, "operationResourceName"> = {
+  apiOrigin: frameworksOrigin,
+  apiVersion: API_VERSION,
+  masterTimeout: 25 * 60 * 1_000,
+  maxBackoff: 10_000,
+};
+
+/**
+ * Creates Stack object from long running operations.
+ */
```

Comment:
nit* Personally, I'd prefer:

1. keeping the HTTP client for Frameworks in `/gcp` like the rest of the http client for GCP apis.

2. Leaving the responsibility of instantiating the poller at the client side. That way, the client can control the poller's configuration like timeout and backoff that's appropriate for the command. (as is done in src/init/features/composer/repo.ts)

Any particular reason you wanted to deviate from the convention and set the structure like this?

Reply:
In case of `src/init/features/composer/repo.ts` it uses `gcp` api's. But I think the `frameworksAPI` is different from `/gcp` family. WDYT?

Reply:
all firebase apis are part of gcp.